### DISCUSSION TITLE
Any timeline for developing on Apple Silicon Macs?

### DISCUSSION THREAD

**Discussion by gutley** [mozilla-mobile/firefox-ios#9340] [General]
What is actually holding up being able to develop on AS Macs? The readme file is bit terse on the subject:

> Please note: due to dependency issues, development of Firefox-iOS is currently only supported on Intel based Macs, and not Apple Silicon based Macs.

This is going to become more and more problematic as time goes by.

Are there any workarounds for someone without access to an Intel Mac?

**Comment by adudenamedruby** [comment]
Heya,

Our big issue is an internal tool called Application Services. Everything else is more or less ready to go for Apple Silicon. I know that our AS team is working hard on making the transition.

There may be workaround (such as building AS locally with your own RUST env) but I haven't done much testing on that so I, unfortunately, can't offer much in the way of help or guidance.

As the owner of an M1, believe me, I am definitely supporting the transition.

**Reply by gutley** [reply]
Thanks for the info. On a related note, are they any issues with building with Monterey on an Intel mac? If so I'll wait to update until it's ok.

**Reply by lmarceau** [reply]
@gutley We didn't try Monterey yet. Will be done in the coming weeks, but for now we're on Big Sur.

**Reply by gutley** [reply]
@lmarceau - Any news on either Apple Silicon or Monterey?

**Reply by lmarceau** [reply]
@gutley No update sorry! One of the big changes we're looking forward to do is Cartage removal, which will enable us to use M1 macs. I guess Monterey testing will be next.

**Reply by gutley** [reply]
> One of the big changes we're looking forward to do is Cartage removal

Indeed, that will be a joyous day when it comes. One less package manager to go wrong for our fork (we already have SPM and Cocoapods).

**Reply by lmarceau** [reply]
@gutley I just learned Monterey is good to go! Didn't realize one of our team members started using it not so long ago. So if you're looking to upgrade, it would be a go on our side.
